_includes/comparison.html

```diff
@@ -63,6 +68,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>any</td>
         <td>any</td>
         <td><a href="https://release-v0-11.docs.openservicemesh.io/docs/guides/traffic_management/ingress/" target="_blank">prepared config for Contour</a>, compatible with any other</td>
+        <td>any</td>
```

Comment:
Cilium includes a full Ingress controller in addition to support any third party Ingress:
https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/ingress/